### ISSUE TITLE
Watcher not launching: Issue + Resolution

### ISSUE THREAD

**Issue by cameronj86** [RundownRhino/aw-watcher-mpv-sender#8]
**TL;DR**: Logger works fine but I could never get the watcher to work until today. Turns out the file ./aw-watcher-mpv needed execute permissions. Not sure if this is just a quirk of Debian or whether edits to the watcher during the last release reverted the file permissions.

#### Successful logger:
```
cat ~/.config/mpv/mpv-history/2023-11-12.log 
1   │ {"time":"2023-11-12T00:15:41Z","filename":"ExampleFile.mp4","kind":"playing","title":"ExampleFile.mp4"}
```
#### aw-watcher-mpv install location
```
locate aw-watcher-mpv$ -r
/home/cameronj86/.local/bin/activitywatch/aw-watcher-mpv/aw_watcher_mpv_sender
```
#### Confirming same location as activitywatch:
```
locate aw-qt$ -r
/home/cameronj86/.cache/activitywatch/log/aw-qt
/home/cameronj86/.config/activitywatch/aw-qt # this is install location
/home/cameronj86/.local/bin/activitywatch/aw-qt
```

#### Original unzipped file just to show the default permissions after d/l
```
ls -l ~/Downloads/Original_Unzipped/aw-watcher-mpv
.rw-r--r--
```
#### Adjustments that I made which resolved my issue
`chmod 744 /home/cameronj86/.local/bin/activitywatch/aw-watcher-mpv/aw-watcher-mpv`

**Comment by RundownRhino** [comment]
Interesting, thanks! Indeed, when I unpack the artifact zips of the latest release, the executable ends up without an x permission. It seems to be the case in the old releases, too, so I don't think it's a regression.
When I do the build process on my computer, the executable pyinstaller writes has the execute permission, and it's preserved when archiving. So I guess it's some weirdness in the CI build specifically (e.g. perhaps `upload-artifact@v3` nukes the file permissions when making the archive). I'll see if I can fix that somehow.

**Comment by cameronj86** [comment]
Sounds good. 🤘🏾
